Sources/SKCore/BuildSystemDelegate.swift

```diff
@@ -0,0 +1,26 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LanguageServerProtocol
+import Basic
+
+/// A delegate which handles various events from the build system.
+///
+/// At the moment this is limited to:
```

Comment:
I would remove this comment, and change the first sentence to something like "A delegate to handle build system events, such as file build settings changes."

Reply:
Done